docs-new/docs/components/lifecycle-operator/deployment-flow.md

```diff
@@ -165,22 +168,27 @@ Additional phases/states exist,
 such as those that describe what happens when something fails.
 
 Whenever something in the system happens (we create a new resource, etc.)
-a Kubernetes event is generated.
+an event is generated.
 The following events are defined as part of Keptn
 but they are not part of the deployment flow.
 These include:
 
 ```shell
-CreateEvaluation
 ReconcileEvaluation
```

Comment:
Are these events documented anywhere?  I guess they are kind of self-documenting  -- I can make a good guess as to what "CreateApp" does but what events like "ReconcileTask" and "ReconcileWorklad" mean is less obvious.  It would be nice to link to the explanation -- it could even be in the code.

Reply:
Reconciliation is an action made by controllers which are part of k8s operators when re-evaluating the state of the resource. It's basically different word for re-evaluation used in k8s.

Reply:
I think maybe the problem is in the description of the section and the title also. The title is misleading, in what sense the events are not part of the deployment flow? A workload version creation is part of te deployment cycle...  Maybe it is worth to separate them in creation deletion of resources and events signaling a new reconciliation loop for a resource

Reply:
Adapted

Reply:
sounds waay more understandable now  👍